docs/02.API-REFERENCE.md

```diff
@@ -3787,11 +3787,13 @@ is no longer needed.
 jerry_value_t
 jerry_exec_snapshot (const uint32_t *snapshot_p,
                      size_t snapshot_size,
+                     size_t func_index,
```

Comment:
I'd rather add a new API to avoid breaking existing projects.
`jerry_exec_snapshot_at_index`?